internal/app/app.go

```diff
@@ -79,11 +79,18 @@ type FileChange struct {
 	Patch       string `json:"patch"`
```

Comment:
Please provide documentation comments for `PRReviewMap` and `PRCommentInfo` structures to improve code maintainability and understandability.